CONTRIBUTING.md

```diff
@@ -42,7 +42,7 @@ Some examples of types of pull requests that are immediately helpful:
 Guidelines for any code contributions:
 
   1. Any significant changes should be accompanied by tests. The project already
-     has good test coverage, so look at some of the existing tests if you're
+     has good test coverage, so look at some existing tests if you're
```

Comment:
This wasn‘t wrong and sounded better before imho. Otherwise "look at _the_ existing tests“ but "some existing tests" sounds awkwardly non-specific